lib/util/utils.ts

```diff
@@ -571,13 +561,40 @@ export function isUrlEncoded(str: string): boolean {
   return str !== decodeURIComponent(str)
 }
 
+export interface Properties {
+  [name: string]: Model
+}
+
+export interface Ref {
+  readonly $ref: string
```

Comment:
>$ref [](start = 11, length = 4)

there is no problem in accessing this is a property (i.e. not as a dictionary key) ?

Reply:
actually seems fine 

---
In reply to: [194477251](https://github.com/Azure/oav/pull/257#discussion_r194477251) [](ancestors = 194477251)